hack/dockerfiles/gosec.Dockerfile

```diff
@@ -0,0 +1,5 @@
+# syntax=docker/dockerfile:1.3
+
+FROM securego/gosec AS validate
+WORKDIR /go/src/github.com/moby/buildkit
+RUN --mount=target=/go/src/github.com/moby/buildkit ./hack/validate-gosec check
```

Comment:
nit: could be either way but as this only calls a single command you can just call gosec in here.

Reply:
As you already set `WORKDIR`, this can be `--mount=target=.`

Reply:
Done.